docs/hacking.md

```diff
@@ -37,3 +37,9 @@ $ export GOPASS_HOMEDIR=/tmp/gp1
 $ mkdir -p $GOPASS_HOMEDIR
 $ go build && ./gopass setup --crypto age --storage gitfs
 ```
+
+## Extending gopass
+
+The main extension model small binaries that use the [gopass API](https://pkg.go.dev/github.com/gopasspw/gopass/pkg/gopass/api) package. This package provides a small and easy to use API that should work with any up to date gopass setup.
+
+We don't have extensive documentation for this, yet. But the [gopass-hibp](https://github.com/gopasspw/gopass-hibp/blob/master/main.go) binary should provide an easy example that can be used as a blueprint.
```

Comment:
Looking at this example, I don't really understand how to "connect" to my Gopass neither. 
Is it just running Gopass as is under the hood, using the same Env variable and same config and config folder as the Gopass binary?

We might want to precise how this "works together" somewhere I think.

Reply:
Should be better now.